### DISCUSSION TITLE
An error occurred. (Error code: 9) (Extra code: 0) Learn more Version 123.0.6312.133 (Official Build) (arm64)

### DISCUSSION THREAD

**Discussion by m4th3r0b0t** [Alex313031/Thorium-MacOS#47] [Show and tell]
![image](https://github.com/Alex313031/Thorium-MacOS/assets/74017484/ba6b8035-13d8-4203-8d80-f53149da5854)
How to fix this? I am using Macbook Pro m1

**Comment by midzer** [comment]
This is a known issue since few versions from upstream. We're looking to fix it.

**Reply by m4th3r0b0t** [reply]
Alright mate. Looking forward to it.
